linkerd/tracer/zipkin/src/main/scala/io/buoyant/linkerd/tracer/ZipkinTracerInitializer.scala

```diff
@@ -20,15 +28,52 @@ case class ZipkinConfig(
 
   @JsonIgnore
   override def newTracer(): Tracer = new Tracer {
-    private[this] val underlying: Tracer =
-      ZipkinTracer.mk(
-        host = host.getOrElse("localhost"),
-        port = port.getOrElse(9410),
-        sampleRate = sampleRate.map(_.toFloat).getOrElse(Sampler.DefaultSampleRate)
+    private[this] val underlying: Tracer = {
+      val transport = ClientBuilder()
+        .name("zipkin-tracer")
+        .hosts(new InetSocketAddress(host.getOrElse("localhost"), port.getOrElse(9410)))
+        .codec(ThriftClientFramedCodec())
+        .reportTo(ClientStatsReceiver)
+        .hostConnectionLimit(5)
+        // using an arbitrary, but bounded number of waiters to avoid memory leaks
+        .hostConnectionMaxWaiters(250)
+        // somewhat arbitrary, but bounded timeouts
+        .timeout(100.millis)
+        .daemon(true)
+        .noFailureAccrual
+        .failFast(false)
+        .build()
```

Comment:
it might be good to add comments indicating which settings differ from the default values from ZipkinTracer